ml-agents/mlagents/trainers/tests/simple_test_envs.py

```diff
@@ -49,8 +49,10 @@ def __init__(
         vec_obs_size=OBS_SIZE,
         var_len_obs_size=VAR_LEN_SIZE,
         action_sizes=(1, 0),
+        gail=False,
     ):
         super().__init__()
+        self.gail = gail
```

Comment:
Can you merge the latest `main` and add this to the seed tuple here https://github.com/Unity-Technologies/ml-agents/blob/65d1983d053e28f2870bc9b1f2aba5d1e1aff218/ml-agents/mlagents/trainers/tests/simple_test_envs.py#L73-L84 ?